### ISSUE TITLE
Stonks is down

### ISSUE THREAD

**Issue by nickwales** [marcopaganini/quotes-exporter#6]
I just tried installing for the first to find the backend API is down at the moment. Having not been aware of any of this until this week I don't know if this is a transient issue or that service is no longer working.

**Comment by marcopaganini** [comment]
Oh yeah, I just noticed it too :( Unfortunately it's been up and down for a
while and now it's down for good. I've been looking for alternatives but
they're all expensive or limited. Most would require users to register
their own APIs, which most won't do :(

If you have a suggestion, please let me know. I'd like to fix this but so
far haven't found any reasonable solution.

**Comment by nickwales** [comment]
From my brief research it looks like everything used yahoo which from what I've read is no longer available. I'll let you know if I find anything!

**Comment by marcopaganini** [comment]
Yeah, yahoo was my next stop, but I soon realized they were not working anymore.

Here's a few options that I've tried or considered:

* Yahoo : Doesn't work anymore.
* Stonks: Doesn't work anymore.
* alphavantage.co: The free API has a ridiculous limit of 25 calls per day.
* polygon.io: Free API allows 5 calls per minute, which is OKish, but only provides end of day data :(
* finnhub.io: 60 API calls per minute, but the offer is very confusing. Need to take more look into this.
* marketstack.com: 1000 requests per month, but only end of day data.
* twelvedata.com: Uses an API credit system. Unclear to me how many requests this would translate to.
